src/Components/FormCreation/FormSection.js

```diff
@@ -64,7 +64,8 @@ function FormSection({
   sectionData,
   questions,
```

Comment:
Hmm, a bit unrelated to this diff, but I think sectionsData contains the section questions arleady, so we don't need to pass this prop in (I could be wrong about what questions is here).

Reply:
Will investigate!